fred split (#26)
SpecialeBajs/speciale-rapport: article/Method/alpha-k-layer-combination.tex

    \caption{Example of the changed effect to $\alpha_k$. }
    \label{tab:alpha-example}
\end{table}

\paragraph{Layer effect based on performance} \label{fredsplit}

> After the experiments done on node degree [ref her tak] it was decided to make an algorithm that chose the layers' effect based on how they perform compared to each other.

Reviewer: ```suggestion
After the experiments done on node degree \autoref{subsec:degree-experiment} it was decided to make an algorithm that chose the layers' effect based on how they perform compared to each other.
```

It was clearly seen that some layers performed substantially better than others and therefore the worst layers might have a negative effect on the final embedding.
\\

> The basic idea of this algorithm is to look at the performance of the individual layers, then based on how many percent worse they performe compared to the best layer they will get a penalty in their effect and the other layers will get an increase.

Reviewer: ```suggestion
The basic idea of this algorithm is to look at the performance of the individual layers, then based on how many percent worse they perform compared to the best layer they will get a penalty in their effect and the other layers will get an increase.
```

Given four layers each layer will have 25\% effect on the final embedding at the start.
The worst layer performs 15\% worse than the best layer, we therefore remove 15 from 25 to get its final effect of 10.

> Meanwhile the 15 is split evenly among the other 3 layers who now have a effect of 30\%.

Reviewer: ```suggestion
Meanwhile, the 15 is split evenly among the other 3 layers who now have an effect of 30\%.
```

> The same process is then repeated but the worst performing layer from the previous iteration is excluded until only on layer remains.

Reviewer: Denne sætning forstår jeg ikke helt. Forstår godt at de bliver ekskluderet, men forstår ikke "until only one layer remains"

> This can result in one or more layers having 0 effect in the end.

Reviewer: ```suggestion
This can result in one or more layers having 0 \% effect in the end.
```

> A more balanced version can be done by not excluding the worst performing layer from the previous iteration.

Reviewer: ```suggestion
A more balanced version can be done by not excluding the worst-performing layer from the previous iteration.
```

    \caption{Example of the changed effect to $\alpha_k$. }
    \label{tab:alpha-example}
\end{table}

\paragraph{Layer effect based on performance} \label{fredsplit}

> After the experiments done on node degree \autoref{subsec:degree-experiment} it was decided to make an algorithm that chose the layers' effect based on how they perform compared to each other.

Reviewer 2: ```suggestion
Based on the results from the node degree experiments seen in \autoref{subsec:degree-experiment}, we decided to make an algorithm that calculated each layers' effect based on how they performed compared to each other.
```

> It was clearly seen that some layers performed substantially better than others and therefore the worst layers might have a negative effect on the final embedding.

Reviewer 2: ```suggestion
It could clearly be seen that some layers performed substantially better than others and therefore the worst layers might have a negative effect on the final embedding.
```

\\

> The basic idea of this algorithm is to look at the performance of the individual layers, then based on how many percent worse they perform compared to the best layer they will get a penalty in their effect and the other layers will get an increase.

Reviewer 2: ```suggestion
The algorithm looks at the performance of each individual layers and then based on how many percent's worse they perform compared to the best layer they will get a penalty in their effect and the other layers will get an increase.
```

> Given four layers each layer will have 25\% effect on the final embedding at the start.

Reviewer 2: Er resten af afsnittet ikke skrevet ud fra figurerne, men refererer kun til dem til sidst ?
